src/lib/components/chat/MessageInput/Commands/Prompts.svelte

```diff
@@ -22,9 +22,18 @@
 	let selectedPromptIdx = 0;
 	let filteredPrompts = [];
 
-	$: filteredPrompts = $prompts
-		.filter((p) => p.command.toLowerCase().includes(command.toLowerCase()))
-		.sort((a, b) => a.title.localeCompare(b.title));
+	$: {
+		if (command && command.length > 1) {
```

Comment:
```suggestion
		if (command && command.length > 0) {
```

Fixes the issue with the slash
<img width="1384" alt="Screenshot 2025-05-22 at 23 38 52" src="https://github.com/user-attachments/assets/3b284166-9963-46e4-a374-146ff9fa2ced" />

Reply:
Thank you great catch